index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="index.css" type="text/css">
+    <title>Gideon Spektor</title>
+</head>
+<body>
+    <nav>
+        <a href="file:///Users/academy/Desktop/portfolio/index.html"><img src="logo.png"></a>
```

Comment:
dont use this link in here, it wont work on anyone elses computer, just replace with a #

Reply:
also you forgot your / at the end of your img self closing tag